src/export/export-chakra.ts

```diff
@@ -130,51 +136,86 @@ const processTextStyles = (
   return updatedTextStyles;
 };
 
+// Reshaping the tokens to match Chakra's theme format (https://chakra-ui.com/docs/styled-system/theme)
+// This function is recursive so that it can climb throughout the whole
+// tree of tokens and make the nesisary undates to every individual token.
+const reshapeDesignTokens = (tokens: Tokens): ChakraTokens => {
+  const newTokens = { ...tokens };
+
+  for (const key in newTokens) {
+    // Deleting the $type keys as Chakra doesn't need them.
+    if (key === '$type') delete newTokens[key];
+
+    // Flattening the `[key]: { $value: ... }` to `[key]: ...`
+    if (newTokens[key]?.$value) {
+      newTokens[key] = newTokens[key].$value;
+    }
+
+    // The only keys that are arrays are shadows, so we convert them to CSS shadows like Chakra expects.
+    if (Array.isArray(newTokens[key])) {
+      newTokens[key] = convertShadowsDesignTokenToCss(newTokens[key]);
+    }
+
+    // If the current key is an object recursively call the function again to climb the tree.
+    if (newTokens[key] !== null && typeof newTokens[key] === 'object') {
+      reshapeDesignTokens(newTokens[key]);
```

Comment:
Since we updated `reshapeDesignTokens` to no longer mutate the passed argument, we'll also need this update this line to:

```
newTokens[key] = reshapeDesignTokens(newTokens[key]);
```